snap/implicit_test.go

```diff
@@ -56,7 +56,7 @@ func (s *InfoSnapYamlTestSuite) TestAddImplicitSlotsOnClassic(c *C) {
 	c.Assert(info.Slots["unity7"].Interface, Equals, "unity7")
 	c.Assert(info.Slots["unity7"].Name, Equals, "unity7")
 	c.Assert(info.Slots["unity7"].Snap, Equals, info)
-	c.Assert(info.Slots, HasLen, 25)
+	c.Assert(info.Slots, HasLen, 26)
```

Comment:
Since moving to implicitSlots, above, should up TestAddImplicitSlotsOutsideClassic length to 16.